packages/desktop/src/renderer/components/widgets/channels/BasicMessage.tsx

```diff
@@ -126,33 +141,38 @@ export const BasicMessageComponent: React.FC<BasicMessageProps> = ({
             justify='flex-start'
             alignItems='flex-start'
             wrap={'nowrap'}>
-            <Grid item className={classes.avatar}>
+            <Grid
+              item
+              className={classNames({ [classes.avatar]: true })}>
               <div className={classes.alignAvatar}>
-                <Jdenticon size='32' value={messageDisplayData.nickname} />
+                {infoMessage ? (
+                  <Icon src={information} className={classes.infoIcon} />
+                ) : (
+                  <Jdenticon size='32' value={messageDisplayData.nickname} />
+                )}
               </div>
             </Grid>
             <Grid container item direction='row'>
               <Grid container item direction='row' justify='space-between'>
-                <Grid
-                  container
-                  item
-                  xs
-                  alignItems='flex-start'
-                  wrap='nowrap'
-                >
+                <Grid container item xs alignItems='flex-start' wrap='nowrap'>
                   <Grid item>
                     <Typography
                       color='textPrimary'
                       className={classNames({
                         [classes.username]: true,
                         [classes.pending]: pending
                       })}>
-                      {messageDisplayData.nickname}
+                      {!infoMessage ? messageDisplayData.nickname : 'Quiet'}
```

Comment:
Could you use `infoMessage ?  'Quiet' : messageDisplayData.nickname` instead of using negation? It's easier to read